embedded-graphics/src/fonts/font_builder.rs

```diff
@@ -165,7 +166,7 @@ where
     }
 }
 
-impl<'a, C, Conf> IntoIterator for &'a FontBuilder<'a, C, Conf>
+impl<'a, 'b, C, Conf> IntoIterator for &'b FontBuilder<'a, C, Conf>
```

Comment:
@jamwaffles this was also an important change to get the &Self trait bounds working. It looks like the reference in this trait implementation had its lifetime tied to the lifetime of the string reference inside FontBuilder; however, in order to implement the `Drawable` trait, those lifetimes needed to be made independent.